### ISSUE TITLE
Binder bot not working for pull requests from forks

### ISSUE THREAD

**Issue by Beramos** [UGentBiomath/COVID19-Model#55]
It seems that the GitHub workflow to generate binder links does not work for pull requests from forks. @stijnvanhoey, I originally thought this was not going to be a problem but now I realise that all changes will be coming from other forks. I'll check on this if I find some time next week

**Comment by stijnvanhoey** [comment]
Indeed, good point. Moreover, the step of `pip intall -e .` should be included as well during setup to make sure the package/model itself is available inside the notebooks...

**Comment by stijnvanhoey** [comment]
The latter issue on the `pip install -e` we can probably fix by adding a `postBuild` configuration file inside binder folder, see [binder docs](https://mybinder.readthedocs.io/en/latest/config_files.html?highlight=postBuild#postbuild-run-code-after-installing-the-environment)

**Comment by Beramos** [comment]
I'm temporarily disabling the binder bot and moving the pip-install to a separate issue #59
